Algorithm.Framework/Alphas/CompositeAlphaModel.cs

```diff
@@ -43,6 +44,24 @@ public CompositeAlphaModel(params IAlphaModel[] alphaModels)
             _alphaModels = alphaModels;
         }
 
+        public CompositeAlphaModel(PyObject[] alphaModels)
```

Comment:
can python make use of `params` keywords? If not it's OK since *not* having `param` likely reduces the possibility for method ambiguity.

Reply:
Python can use keyword parameters.  When the keyword is omitted but a value is passed nonetheless, the keyword inherits the positional parameter.  It's bad style to not expressly use named keyword parameters when they exist, because obviously, if the position of the keywords changes with a change to the method definition, then the invoker will no longer function as intended...

Reply:
@disruptek , `CompositeAlphaModel` is not a python object and we (pythonnet) have an issue on keyword/names parameters:
https://github.com/QuantConnect/Lean/issues/1696